packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -74,33 +82,42 @@ function optionalContent(copy) {
  *
  * @param {object} cta cta object
  * @param {string} type type of card (clickable/static)
- * @param {string} href url for card
- * @param {object} Icon cta icon for card
  * @returns {object} JSX object
  */
-function renderFooter(cta, type, href, Icon) {
+function renderFooter(cta, type) {
   return (
     <footer className={`${prefix}--card__footer`}>
-      {type !== 'link' ? (
-        <CTA style="text" type={cta.type} href={href} copy={cta.copy} />
-      ) : (
-        Icon && <Icon />
-      )}
+      {cta &&
```

Comment:
Only suggestion is that you return null on the footer if cta isn't there.

Reply:
done